examples/harvest/Dockerfile

```diff
@@ -18,6 +18,11 @@ RUN apk add --no-cache \
         openssl-dev \
         python3-dev 
 
+RUN pip install -U pip
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+
+ENV PATH="/root/.cargo/bin:${PATH}"
+
```

Comment:
Instead of installing it like that, can you please add the rust and cargo packages as part of the apk packages to be installed, just after line 19?

Reply:
thank you for the feedback, I adapted it accordingly and the docker-compose-build runs through.